app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: list[dict]) -> list:
+    people_dict = {}
```

Comment:
The Person.people dictionary is being updated every time a Person instance is created. This means that you can directly access the Person.people dictionary inside your create_person_list function without needing the local people_dict.

So, you can remove the people_dict and use Person.people instead, simplifying your code.